lib/iris/fileformats/netcdf/loader.py

```diff
@@ -719,18 +719,19 @@ def set(
             Kwargs specifying chunksizes for dimensions of file variables.
             Each key-value pair defines a chunksize for a named file
             dimension, e.g. {'time': 10, 'model_levels':1}.
+            Values of -1 will lock the chunk to the size of it's shape.
```

Comment:
```suggestion
            Values of ``-1`` will lock the chunk to the size of its shape.
```

Please could we take a general look at this this docstring for some fine points:

- monospacing
- [single backticks](https://numpydoc.readthedocs.io/en/latest/format.html#parameters)
- how we are supposed to do `str --> int`
- documenting the default value or optionality

[Style guide — numpydoc v1.7.0rc0.dev0 Manual](https://numpydoc.readthedocs.io/en/latest/format.html)

Reply:
str --> int:
I couldn't find anything on this, perhaps I just missed it. The numpy docs only offer `var: dict`, and that's it.